target/linux/ramips/image/mt7621.mk

```diff
@@ -1680,14 +1680,15 @@ TARGET_DEVICES += netgear_wndr3700-v5
 
 define Device/netis_wf2881
   $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   FILESYSTEMS := squashfs
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 129280k
   UBINIZE_OPTS := -E 5
   UIMAGE_NAME := WF2881_0.0.00
-  KERNEL_INITRAMFS := $(KERNEL_DTB) | netis-tail WF2881 | uImage lzma
+  KERNEL_INITRAMFS := $$(KERNEL) | netis-tail WF2881
```

Comment:
Why do you change the `KERNEL_INITRAMFS`? 
Did you test initramfs booting?

Reply:
Original commit be3e98ce26da5e737744427e1d59dd1c0508edef was merged by pull request [#2763](https://github.com/openwrt/openwrt/pull/2763)

Tested initramfs booting from stock web shell and sysupgrade on initramfs booting.